R/to_xml.R

```diff
@@ -45,7 +45,11 @@ to_xml <- function(path, encoding = "UTF-8", sourcepos = FALSE, anchor_links = T
     xml2::read_xml(encoding = encoding) -> body
 
   parse_rmd(body)
-  body <- protect_tickbox(body, md_ns())
+  if (utils::packageVersion("commonmark") < "1.8.0") {
+    #nocov start
+    body <- protect_tickbox(body, md_ns())
```

Comment:
as it's one line maybe rather just use `# nocov` at the end of it?

Reply:
😅